win32/service.py

```diff
@@ -134,7 +134,12 @@ def _update_config_file(self, config):
         config.update(registry_conf)
         if registry_conf:
             log.info('Updating conf file options: %s', registry_conf.keys())
-            update_conf_file(registry_conf, get_config_path())
+            try:
+                update_conf_file(registry_conf, get_config_path())
+                log.info('update succeeded, deleting old values')
+                remove_registry_conf()
```

Comment:
I wanted to remove the values from the registry so we don't end up with "two masters"... you put something on the install config.  Then the install works.  Then you change the config, but this is still lingering in the registry, waiting to be written back out to the config file (and killing your changes)